modular_skyrat/modules/cargo/code/packs.dm

```diff
@@ -119,6 +119,20 @@
 					/obj/item/melee/classic_baton/peacekeeper)
 	crate_name = "baton crate"
 
+/datum/supply_pack/security/croonsurplus
+	name = "Croon Weapons Crate"
+	desc = "Help out in a local revolt, or fund a civil war, it's not like you have a choice in supplier. (Does not include additional ammo)"
```

Comment:
I feel like this is worded a little oddly, suggests to me somewhat that it comes without ammo. Or it could at least be misread as saying that very easily

Reply:
clarified